doc/source/whatsnew/v1.2.1.rst

```diff
@@ -15,9 +15,9 @@ including other versions of pandas.
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
 - The deprecated attributes ``_AXIS_NAMES`` and ``_AXIS_NUMBERS`` of :class:`DataFrame` and :class:`Series` will no longer show up in ``dir`` or ``inspect.getmembers`` calls (:issue:`38740`)
-- :meth:`to_csv` created corrupted zip files when there were more rows than ``chunksize`` (issue:`38714`)
-- Fixed a regression in ``groupby().rolling()`` where :class:`MultiIndex` levels were dropped (:issue:`38523`)
-- Bug in repr of float-like strings of an ``object`` dtype having trailing 0's truncated after the decimal (:issue:`38708`)
+- Fixed regression in :meth:`to_csv` that created corrupted zip files when there were more rows than ``chunksize`` (:issue:`38714`)
+- Fixed regression in ``groupby().rolling()`` where :class:`MultiIndex` levels were dropped (:issue:`38523`)
```

Comment:
this may cause a conflict (groupby.rolling) as haven't actually merged that change in 1.2.x  as still discussing. i suppose we should revert it.

Reply:
good point. I won't touch that entry in this PR. was just extra tidying while fixing line above.

Reply:
kk cool, yeah if revert that line then no objection